test/e2e/config/azure-dev.yaml

```diff
@@ -46,8 +46,8 @@ providers:
           new: --metrics-addr=:8080
       files:
         - sourcePath: "../data/shared/v1alpha4/metadata.yaml"
-    - name: v1.2.6
-      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.2.6/bootstrap-components.yaml
+    - name: v1.3.0
+      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.3.0-rc.1/bootstrap-components.yaml
```

Comment:
since the actual v1.3.0 release dropped today, should we skip the rc and directly bump to v1.3.0?

Reply:
Done